packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -3191,9 +3191,7 @@ abstract final class CupertinoIcons {
   /// This is the same icon as [create_solid] which is available in cupertino_icons 0.1.3.
   static const IconData square_pencil = IconData(0xf417, fontFamily: iconFont, fontPackage: iconFontPackage);
   /// <i class='cupertino-icons md-36'>square_pencil_fill</i> &#x2014; Cupertino icon named "square_pencil_fill". Available on cupertino_icons package 1.0.0+ only.
-  /// This is the same icon as [create] which is available in cupertino_icons 0.1.3.
-  /// This is the same icon as [create_solid] which is available in cupertino_icons 0.1.3.
-  static const IconData square_pencil_fill = IconData(0xf417, fontFamily: iconFont, fontPackage: iconFontPackage);
+  static const IconData square_pencil_fill = IconData(0xf418, fontFamily: iconFont, fontPackage: iconFontPackage);
```

Comment:
codepoint change: https://github.com/xster/framework7-icons/pull/2/files#diff-8d861fabe46d688163d4afa524349547c56cea98546a41addd7c4f5c21eece72R55

Reply:
There're new assertions in the script that makes sure that there are no codepoint conflicts so 0xf418 is not occupied by another symbol.